samples/defaultauth.js

```diff
@@ -24,19 +24,20 @@ var compute = google.compute('v1');
  * method rather than creating your own JWT or Compute client directly.
  *
  * Note: In order to run on a local developer machine, it is necessary to download a private key
- * file to your machine, and to set a local environment variable pointing to the location of the
+ * file to your machine, and to set a local environment constiable pointing to the location of the
```

Comment:
I'm not a native English speaker but it looks like replacing all `var` with `const` caused some unexpected consequences!

Reply:
*laughs* oops.  Nice catch.